Example/ReusableDemo/CollectionViewCells/MyXIBIndexSquaceCell.swift

```diff
@@ -20,11 +20,11 @@ import Reusable
  * Which in fact is just a convenience protocol that combines
  * both `NibLoadable` + `Reusable` protocols.
  */
-class MyXIBIndexSquaceCell: UICollectionViewCell, NibReusable {
-  @IBOutlet private weak var sectionLabel: UILabel!
-  @IBOutlet private weak var rowLabel: UILabel!
+final class MyXIBIndexSquaceCell: UICollectionViewCell, NibReusable {
+  @IBOutlet fileprivate weak var sectionLabel: UILabel!
+  @IBOutlet fileprivate weak var rowLabel: UILabel!
 
-  func fill(indexPath: NSIndexPath) {
+  func fill(_ indexPath: IndexPath) {
```

Comment:
There's an [`IndexPath` typealias](https://github.com/phatblat/Reusable/blob/ben/swift3/Sources/View/IndexPath.swift#L12-L14) which defines this missing type only for Swift 2.